### PR TITLE
Dependencies update

### PR DESCRIPTION
**Pull request by AndreHaueisen**
Describe the purpose of the pull request.

#### Fixes / Enhancements

- Updated dependencies

#### Docs

- Updated changelogs

### CODE REVIEW

File: packages/graphql/pubspec.yaml [hunk 1]
```diff
@@ -1,30 +1,34 @@
...
   pedantic: ^1.8.0+1
   mockito: ^4.0.0
   test: ^1.5.3
-  test_coverage: ^0.3.0+1
-  http_parser: ^3.1.4
+  test_coverage: ^0.5.0
+  http_parser: ^4.0.0
+
 environment:
   sdk: '>=2.6.0 <3.0.0'
```
Comment (b-jan) [inline packages/graphql/pubspec.yaml:31]: Tests failed: It requires SDK version >=2.12.0-0 <3.0.0

File: packages/graphql_flutter/pubspec.yaml [hunk 2]
```diff
@@ -1,24 +1,31 @@
...
-  http: ^0.12.1
+  test: ^1.16.5
+  http: ^0.13.0
+
+dependency_overrides:
+  source_span: ^1.8.1
+  gql: ^0.13.0-nullsafety.1
+
 environment:
   sdk: '>=2.6.0 <3.0.0'
```
Comment (b-jan) [inline packages/graphql_flutter/pubspec.yaml:31]: Tests failed: It requires SDK version >=2.12.0-0 <3.0.0

Comment (AndreHaueisen) [reply packages/graphql_flutter/pubspec.yaml]: Is there any chance we can update the flutter version on CI?

Comment (b-jan) [reply packages/graphql_flutter/pubspec.yaml]: I thought it was automatic but apparently no

**Comment by AndreHaueisen** [comment]
Not sure how to solve this. Closing it

File: packages/graphql_flutter/pubspec.yaml (hunk 2 above)
Comment (b-jan) [reply packages/graphql_flutter/pubspec.yaml]: @AndreHaueisen Can you try updating `cirrusci/flutter:latest-web` in .circleci file

Comment (b-jan) [reply packages/graphql_flutter/pubspec.yaml]: maybe cirrusci/flutter:stable

**Comment by b-jan** [comment]
> Not sure how to solve this. Closing it

@AndreHaueisen Can you try updating `cirrusci/flutter:latest-web` in .circleci file
maybe with `cirrusci/flutter:stable`

**Pull request closed without merging**